build.gradle

```diff
@@ -230,6 +245,37 @@ jar {
         name project.name
         description project.description
         license project.basePom.getModel().getLicenses()[0].url
+        instruction 'Import-Package',
+            'android.*;resolution:=optional',
+            'com.google.appengine.*;resolution:=optional',
+            'com.squareup.okhttp;resolution:=optional',
+            'okhttp3;resolution:=optional',
+            'okio;resolution:=optional',
+            '*'
+        instruction 'Export-Package',
+            'com.dropbox.core;version=3.0.5',
```

Comment:
For clarity, the more idiomatic thing to do nowadays would be to use a `package-info.java` file with the `@Version` OSGi annotation. IMO, however, introducing that into the project would be fairly invasive at this point.